docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,143 @@
+# Pixi Global Manifest
+
+### Motivation
+
+`pixi global` is currently limited to imperatively managing CLI packages.
+The next iteration of this feature should fulfill the following needs:
+
+- Shareable global environments.
+- Allow managing complex environments with multiple packages as dependencies
+- Flexible exposure of binaries
+
+## Design Considerations
+
+There are a few things we wanted to keep in mind in the design:
+
+1. **User-friendliness**: Pixi is a user focussed tool that goes beyond developers. The feature should have good error reporting and helpful documentation from the start.
+2. **Keep it simple**: The CLI should be all you need to interact with the global environments.
+3. **Unsurprising**: Simple commands should still behave similar to traditional package managers.
+4. **Human Readable**: Any file created by this feature should be human-readable and modifiable.
+
+## Manifest
```

Comment:
I think it is also good to add here versioning for our further improvements or iterations
```
[manifest]
version = 1
```

Reply:
If we also plan to have multiple `pixi-global` manifests, should we also add description field or author field in it?

Reply:
if we plan to have lock files, do we need to think if we restrict `pixi-global` per platforms and how it will look / behave?

Reply:
> I think it is also good to add here versioning for our further improvements or iterations
> 
> ```
> [manifest]
> version = 1
> ```

We don't have that for pixi.toml at the moment

I think we should have it either for both or none of them

Reply:
I think it's a good idea, because of creating an error when you can not parse it.

With `pixi.toml` we didn't because having no version is equal to version `0` and having the extra field in a very interactive file felt unnecessary. But we should add it at some point.